script.js

```diff
@@ -44,22 +33,23 @@ function listenForCollisions() {
 //win if u get to Exit door (no timers yet)
 function listenForWinning() {
     document.querySelector('.winningSquare').addEventListener('mouseenter', (e) => {
-        win()
+        // code here to activate win modal displaying
+        winModal.style.display = "block";
         e.stopPropagation();
     })
 }
 
 //call functions from here
 
 let ghost = document.querySelector('.ghost');
-let pumpkin = document.querySelector('.pumpkinImage');
-let reaper = document.querySelector('.grim_reaperImage');
+let pumpkin = document.querySelector('.pumpkin');
```

Comment:
do these need to change? if not, make them constants

Reply:
made consts